uri-schemes.md

```diff
@@ -0,0 +1,233 @@
+# Deltachat URI-Schemes
+
+Currently URI schemes are mainly supported by the app-internal QRcode scanner, at this time only `openpgp4fpr` (used for securejoin) is also registered as uri scheme handler on all platforms ([android](https://github.com/deltachat/deltachat-android), [ios](https://github.com/deltachat/deltachat-ios) and [desktop](https://github.com/deltachat/deltachat-desktop))
+
+- related forum topic: https://support.delta.chat/t/custom-deltachat-url-uri-scheme/346
+- core sourcecode that handles uri schemes: https://github.com/deltachat/deltachat-core-rust/blob/master/src/qr.rs and https://github.com/deltachat/deltachat-core-rust/blob/master/src/securejoin/qrinvite.rs
+
+## **chat.delta**
+
+Generic deltachat scheme, currently used by android as callback url for the oauth2 login method.
+| | |
+| ------------------- | ------------------------------- |
+| Scheme | `chat.delta` |
+| Used for | oauth2 Login (only android currently) |
+| Related Terms\* | - |
+| Availible on | iOS (defined, but not used yet), android |
+| Handled by the core| No|
+| Other apps using it | none, only DeltaChat |
+| Notes | Only used as scheme currently, not in qr codes |
+
+### Syntax
+
+[TODO]
```

Comment:
What this section is for?

Reply:
what the parts an uri with that uri scheme is made of